clients/go/zbc/oauthCredentialsProvider_test.go

```diff
@@ -37,7 +37,24 @@ type mutableToken struct {
 	value string
 }
 
-func TestOAuthCredentialsProvider(t *testing.T) {
+type oauthCredsProviderTestSuite struct {
+	suite.Suite
```

Comment:
We can make a base suite which does the environment clear if that simplifies things, and replace the `suite.Suite` embedded type for that